Remotion/Validation/UnitTests/Implementation/TestDomain/FakeValidationMessage.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Globalization;
+using Remotion.Validation.Implementation;
+
+namespace Remotion.Validation.UnitTests.Implementation.TestDomain
+{
+  public class FakeValidationMessage : ValidationMessage
+  {
+    private readonly string _message;
+
+    public FakeValidationMessage (string message)
+    {
+      _message = message;
+    }
+
+    public override string Format (CultureInfo culture, IFormatProvider formatProvider, params object[] parameters)
+    {
+      throw new NotImplementedException();
```

Comment:
I think we should build this as
return parameters.Aggregate(new StringBuilder(_message), (p, sb) => sb.AppendFormat(" | '{0}')).ToString();

still relatively simple, no exception, all information available.